dll/win32/shell32/brfolder.cpp

```diff
@@ -253,6 +253,73 @@ BrFolder_GetName(
     return bSuccess;
 }
 
+static BOOL
+BrFolder_IsTreeItemInEnum(
+    _Inout_ BrFolder *info,
+    _In_ HTREEITEM hItem,
+    _Inout_ IEnumIDList *pEnum)
+{
+    BrItemData *pItemData = BrFolder_GetItemData(info, hItem);
+    if (!pItemData)
+        return FALSE;
+
+    pEnum->Reset();
+
+    BOOL ret = FALSE;
```

Comment:
Unless you plan to add other stuff after the while-loop later, you might simplify a bit the code as follows:
```suggestion
```
(do as you wish)

Reply:
Fixed.